src/toolkit/gis.h

```diff
@@ -0,0 +1,158 @@
+#ifndef CYCLUS_SRC_GIS_H_
+#define CYCLUS_SRC_GIS_H_
+
+#include <math.h>
+#include <stdio.h>
+#include <sstream>
```

Comment:
The above includes should be moved to the *.cc file so that compile times are reduced. The utilities that they include are not actually used in the header itself.